src/main/java/dan200/computercraft/shared/peripheral/generic/data/ItemData.java

```diff
@@ -116,6 +118,28 @@ private static ITextComponent parseTextComponent( @Nonnull INBT x )
         }
     }
 
+    /**
+     * Retrieve all item groups an item stack pertains to.
+     *
+     * @param stack     Stack to analyse
+     * @return A filled list that contains pairs of item group IDs and their display names.
+     */
+    @Nonnull
+    private static List<Map<String, Object>> getAllItemGroups( @Nonnull ItemStack stack )
+    {
+        ArrayList<Map<String, Object>> groups = new ArrayList<>( 1 );
+
+        for ( ItemGroup g : stack.getItem().getCreativeTabs() )
+        {
+            Map<String, Object> group = new HashMap<>( 2 );
+            group.put( "id", g.getId() );
+            group.put( "displayName", g.getDisplayName().getString() );
```

Comment:
On 1.16, `getDisplayName` is client only. I think we'll need an access transformer here to access the underlying `displayName` field.

Reply:
No longer uses `getDisplayName` or `getId`, just the exposed `langId`